test/jdk/javax/swing/SwingWorker/TestDoneBeforeDoInBackground.java

```diff
@@ -83,25 +86,27 @@ public void propertyChange(PropertyChangeEvent evt) {
                     // Before doInBackground method is invoked,
                     // SwingWorker notifies PropertyChangeListeners about the
                     // property change to StateValue.STARTED
-                    if (doInBackgroundFinished.get() &&
-                         worker.getState() == SwingWorker.StateValue.STARTED) {
+                    if (doInBackgroundFinished.get()
+                        && worker.getState() == SwingWorker.StateValue.STARTED) {
```

Comment:
```suggestion
                    if (doInBackgroundStarted.get()
                        && worker.getState() == SwingWorker.StateValue.STARTED) {
```
Align the condition to the error message.